src/routes/campaigns/analytics/filters/time-filters.ts

```diff
@@ -1,47 +1,88 @@
-import dayjs, { Dayjs } from "dayjs";
+import dayjs from "dayjs";
+import { t } from "@lingui/macro";
 
-interface TimeFilterEntry {
+export interface TimeFilterEntry {
   label: string;
   id: string;
-  from: Dayjs | undefined;
-  to: Dayjs | undefined;
+  from: dayjs.Dayjs | undefined;
+  to: dayjs.Dayjs | undefined;
   divider?: boolean;
 }
 
-export function buildTimeFilters(
-  baseDate: Dayjs = dayjs.utc(),
-  minDate: Dayjs = baseDate.subtract(3, "month"),
-  maxDate: Dayjs = baseDate,
-): TimeFilterEntry[] {
-  const buildFilterForMonth = (dateInMonth: Dayjs) => ({
-    label: dateInMonth.format("MMMM YYYY"),
-    id: dateInMonth.format("YYYY-MM"),
-    from: dateInMonth.startOf("month"),
-    to: dateInMonth.endOf("month"),
-  });
-
-  // build the per-month entries from minDate to maxDate
-  let currentMonthFilter = minDate;
-
-  // never count beyond the "baseDate" (usually "now" in the UI context)
-  const actualMaxDate = dayjs.min(maxDate, baseDate);
-
-  const monthsFilters = [];
-  while (currentMonthFilter.startOf("month").isBefore(actualMaxDate)) {
-    monthsFilters.push(buildFilterForMonth(currentMonthFilter));
-    currentMonthFilter = currentMonthFilter.add(1, "month");
-  }
-
+export function buildTimeFilters(): TimeFilterEntry[] {
   return [
     {
-      // eslint-disable-next-line lingui/no-unlocalized-strings
-      label: "All time",
+      label: t`All time`,
       id: "all-time",
       divider: true,
       from: undefined,
       to: undefined,
     },
-    // TODO: dayjs has a separate localization process, should use it but address in a separate PR
-    ...monthsFilters.reverse(),
+    {
```

Comment:
I think there are too many options here this is overwhelming.

Let's talk about which ones we can drop. My instinct says just:

All Time
Last 7 Days
Last 30 Days
This Month
Last Month
Custom

Why this set?
- 7 days / 30 days just reduces the recency
- This Month and Last Month aligns with invoicing
- and custom for anything else

Yesterday and Today just don't make sense on the default "daily" graph view and show non-sensical data.